data/mods/Aftershock/items/tools.json

```diff
@@ -221,21 +221,11 @@
     "name": { "str": "UPS", "str_pl": "UPS's" },
     "description": "This is a unified power supply, or UPS.  It is a device developed jointly by military and scientific interests for use in combat and the field.  The UPS is designed to power armor and some guns, but drains batteries quickly.  It can be worn around to either leg for ease of access, and it's been waterproofed to protect the delicate electronics.  Has it's own custom battery, with higher capacity and rechargeable, but not removeable",
     "coverage": 5,
-    "ammo": "battery",
-    "initial_charges": 1000,
-    "max_charges": 1500,
+    "ammo_type": [ "battery" ],
```

Comment:
Apparently this does not work for `TOOL_ARMOR` item type.

Reply:
Thats weird, its inclusion did seem important when I tested it, although I might be misremembering.

I'll check again and follow up with a PR if necessary.